tox.ini

```diff
@@ -6,10 +6,18 @@ deps=pytest
      flake8
      flake8-import-order
      mypy
+     bandit
+     pylint
 whitelist_externals=
     sed
+    bash
 commands =
     flake8 src tests setup.py
+    # Skip 'assert used' issue as this is a pytest plugin.'
+    # Use csv format because it has cleanest output.
+    bandit -s B101 -f csv -r src/ tests/
+    # Execution in bash needed for travis
+    bash -c 'pylint src/pytest_workflow/ tests/*.py'
```

Comment:
Is the globbing on tests really required here?

Reply:
Yes. Otherwise it will complain there is no `__init__.py` Alternatively I could add an empty __init__.py.

Reply:
Nvm about that then, was just wondering.